src/VideoManager/VideoReceiver/GStreamer/GStreamer.cc

```diff
@@ -103,28 +104,36 @@ static void _qgcputenv(const QString &key, const QString &root, const QString &p
 
 static void _setGstEnvVars()
 {
-    const QString currentDir = QCoreApplication::applicationDirPath();
-    qCDebug(GStreamerLog) << "App Directory:" << currentDir;
-
-#if defined(Q_OS_MACOS) && defined(QGC_GST_MACOS_FRAMEWORK)
-    _qgcputenv("GST_REGISTRY_REUSE_PLUGIN_SCANNER", "no");
-    _qgcputenv("GST_PLUGIN_SCANNER", currentDir, "/../Frameworks/GStreamer.framework/Versions/1.0/libexec/gstreamer-1.0/gst-plugin-scanner");
-    _qgcputenv("GST_PTP_HELPER_1_0", currentDir, "/../Frameworks/GStreamer.framework/Versions/1.0/libexec/gstreamer-1.0/gst-ptp-helper");
-    _qgcputenv("GIO_EXTRA_MODULES", currentDir, "/../Frameworks/GStreamer.framework/Versions/1.0/lib/gio/modules");
-    _qgcputenv("GST_PLUGIN_SYSTEM_PATH_1_0", currentDir, "/../Frameworks/GStreamer.framework/Versions/1.0/lib/gstreamer-1.0"); // PlugIns/gstreamer
-    _qgcputenv("GST_PLUGIN_SYSTEM_PATH", currentDir, "/../Frameworks/GStreamer.framework/Versions/1.0/lib/gstreamer-1.0");
-    _qgcputenv("GST_PLUGIN_PATH_1_0", currentDir, "/../Frameworks/GStreamer.framework/Versions/1.0/lib/gstreamer-1.0");
-    _qgcputenv("GST_PLUGIN_PATH", currentDir, "/../Frameworks/GStreamer.framework/Versions/1.0/lib/gstreamer-1.0");
-    _qgcputenv("GTK_PATH", currentDir, "/../Frameworks/GStreamer.framework/Versions/1.0");
-#elif defined(Q_OS_WIN)
+    const QString appDir = QCoreApplication::applicationDirPath();
+
+#if (defined(Q_OS_MACOS) && defined(QGC_GST_MACOS_FRAMEWORK)) || defined(Q_OS_WIN)
+    QString bundledGStreamerPath;
+    #if (defined(Q_OS_MACOS) && defined(QGC_GST_MACOS_FRAMEWORK))
+        bundledGStreamerPath = appDir + "/../Frameworks/GStreamer.framework/Versions/1.0/";
+        _qgcputenv("GTK_PATH", bundledGStreamerPath);
+    #elif defined(Q_OS_WIN)
+        bundledGStreamerPath = appDir + "/../";
```

Comment:
@HTRamsey Just realizing my stupidity here. Once you get your thing through could you let me know what I should be checking for existence to understand gstreamer is bundled in? I'm not sure of the Windows install gstreamer setup.